codespell_lib/data/dictionary.txt

```diff
@@ -85,9 +86,12 @@ acccept->accept
 acccepted->accepted
 acccepting->accepting
 acccepts->accepts
+accces->access
 acccess->access
+acccessed->accessed
 acccesses->accesses
 acccessible->accessible
+acccessor->accessors
```

Comment:
the plural here is wrong. `accessor` should be the solution

Reply:
I'd agree in this case, although accessors is also a valid spelling according to Wiktionary (although not Chrome's spellchecker) https://en.wiktionary.org/wiki/accessors#English

Reply:
thanks a lot both for the review. Tell me how to procede.